reference/orders.v2.oas2.yml

```diff
@@ -3256,6 +3378,24 @@ components:
           readOnly: true
           example: /orders/101/consignments
           type: string
+        pickups:
+          description: Pickup consignments.
+          type: array
+          readOnly: true
+        shipping:
+          type: object
+          description: Shipping consignments.
+        downloads:
+          type: object
+          maxItems: 1
+          description: Digital consignments for products that are downloaded.
+        email:
+          type: object
+          description: Email consignments for gift certificates.
+          properties:
+            gift_certificates:
+              type: array
+      x-internal: false
```

Comment:
would it be better to define two different objects and have `consignments` reference both of them with a `oneOf:` or similar?
The api isnt going to return an object with all these properties. It will either be `url` and  `resource` or the new fields

Reply:
I think that would be worth a try. Let's see if it lints.